### DISCUSSION TITLE
Can we branch out to other AWS Regions later, once we have the Ref Arch? Do we need to do special things for it or simply set "region=us-east-1" somewhere in a config file?

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#315] [Help] [answered]
Can we branch out to other AWS Regions later, once we have the Ref Arch? Do we need to do special things for it or simply set "region=us-east-1" somewhere in a config file?

**Comment by yorinasub17** [comment, marked as answer]
See https://github.com/gruntwork-io/knowledge-base/discussions/216#discussioncomment-2232046 for the steps.
